JoonasHeinonen/escape-from-zordoom-prison
# feature/character-movement-and-animation: Created the animations and …

Link to the ticket: [Character movement and animation](https://trello.com/c/KAjunCqH/12-create-a-theme-for-the-game)
![image](https://user-images.githubusercontent.com/33086486/181125589-938017ea-4010-4912-b443-acfa57bfea2f.png)

**Changes:**
 - Animated Angela for the first time.
 - Created image resource for the bolts
 - Initiated the rotation of the sprite.
 - Changed the actual image source to Sprite3D, although the Mesh node is still active.
 - Lighting will effect Angela's sprite.

**How to test:**
 - See that the animation is working perfectly.
 - See that the lighting is consistent.
 - Verify that the sprites are of good quality and of pixel art.

 **Notes and sources:**
  - [Godot Recipes: Animation States](https://www.youtube.com/watch?v=0bq2OIjHxk4&ab_channel=KidsCanCode)
  - [How to animate Sprite Sheets in Godot using AnimationPlayer (beginner tutorial)](https://www.youtube.com/watch?v=FEwE6myyz_I&ab_channel=DrewConley)
  - [Configuring your Godot project for pixel art
](https://www.youtube.com/watch?v=Nsj5SGEUZhU&t=27s&ab_channel=TheShaggyDev)
 - [Images appear with incorrect colors in the editor and in-game](https://godotengine.org/qa/58320/images-appear-with-incorrect-colors-in-the-editor-and-in-game)
